documentation/submitting/submitting.md

```diff
@@ -34,6 +33,17 @@ In order to send the data to the service team, there must be an email address to
 - JSON (TODO)
 - CSV (TODO)
 
+## JSON output
+
+Form submitter can send a webhook to your backend service to process submissions automatically.
+A representation of the users' answers is in JSON format data to a chosen destination.
+This JSON data is encrypted using a shared key via the [JWE standard](https://tools.ietf.org/html/rfc7516).
```

Comment:
might be good to add must be exactly 16 characters long with our current implementation